src/Microsoft.Health.Dicom.Core/Features/Context/DicomRequestContext.cs

```diff
@@ -75,6 +75,8 @@ public class DicomRequestContext : IDicomRequestContext
 
     public long ResponseSize { get; set; }
 
+    public int RequestParts { get; set; }
```

Comment:
How about mulitpart response? WADO can get multiple instances in a study?

Should we have more generic InstanceCount or something like that?

Reply:
Just updated to "PartCount", since we need to count the number of frame for frame WADO requests (as discussed), so "InstanceCount" doesn't make sense anymore